integration_tests_config_template.env

```diff
@@ -0,0 +1,7 @@
+INTEGRATION_RPC_URL=http://example-node-url.com/rpc
```

Comment:
let's rename the file to something like: `integration_tests.env.example`, and add a general info in README about configuring and running those tests